core/issuers.js

```diff
@@ -184,36 +138,19 @@ Description must be truthy: ${description}`;
     return amount;
   }
 
-  const mint = harden({
+  const coreMint = harden({
```

Comment:
It is more like an abstract superclass, but in the instance rather than class domain. It is object-like in the same sense as the object it contributes to. `Functions` I think is the wrong emphasis. `core` is good, in that it is the core object behavior that all derived behaviors are variations on.

In the case on non-customization, this core object essentially is the object being returned. It has exactly the same behavior and differs only in non-observable identity.

In any case, whatever we do here, we should do it consistently for all such object customization.